pkgs/development/haskell-modules/configuration-common.nix

```diff
@@ -950,18 +950,6 @@ self: super: {
   # vector dependency < 0.12
   imagemagick = doJailbreak super.imagemagick;
 
-  # Elm is no longer actively maintained on Hackage: https://github.com/NixOS/nixpkgs/pull/9233.
-  Elm = markBroken super.Elm;
-  elm-build-lib = markBroken super.elm-build-lib;
-  elm-compiler = markBroken super.elm-compiler;
-  elm-get = markBroken super.elm-get;
-  elm-make = markBroken super.elm-make;
-  elm-package = markBroken super.elm-package;
-  elm-reactor = markBroken super.elm-reactor;
-  elm-repl = markBroken super.elm-repl;
-  elm-server = markBroken super.elm-server;
-  elm-yesod = markBroken super.elm-yesod;
```

Comment:
Did you check whether this marks any packages under `elmPackages` as broken by accident? I'm not sure off the top of my head if `configuration-common.nix` gets applied there.